spec/ParseGraphQLServer.spec.js

```diff
@@ -2600,11 +2600,13 @@ describe('ParseGraphQLServer', () => {
               // "SecondaryObject:bBRgmzIRRM" < "SecondaryObject:nTMcuVbATY" true
               // base64("SecondaryObject:bBRgmzIRRM"") < base64(""SecondaryObject:nTMcuVbATY"") false
               // "U2Vjb25kYXJ5T2JqZWN0OmJCUmdteklSUk0=" < "U2Vjb25kYXJ5T2JqZWN0Om5UTWN1VmJBVFk=" false
-              expect(
-                findSecondaryObjectsResult.data.secondaryObjects.edges[0].node.objectId
-              ).toBeLessThan(
-                findSecondaryObjectsResult.data.secondaryObjects.edges[1].node.objectId
-              );
+              if (process.env.PARSE_SERVER_TEST_DB != 'postgres') {
```

Comment:
Unfortunately, this is not how tests are exluded for certain DB adapters, can you please remove all the:

```
if (process.env.PARSE_SERVER_TEST_DB != 'postgres') {
```

There is a `it_only_mongo`, etc. syntax that is used. However, this should not be part of this PR, because this PR only reformats error messages. Changing tests is a delicate thing that needs to be handled in a separate issue and PR.